pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1989,7 +1989,7 @@ protected void handleTcClientConnectRequest(CommandTcClientConnectRequest comman
         });
     }
 
-    private boolean checkTransactionEnableAndSenError(long requestId) {
+    private boolean checkTransactionEnabled(long requestId) {
```

Comment:
It would be confusing for developers if we removed the `SendError` here. Because this method does have the send error process in it

Reply:
Ok, just fix spelling mistakes.